SimDataFormats/Associations/interface/LayerClusterToCaloParticleAssociator.h

```diff
@@ -0,0 +1,50 @@
+#ifndef SimDataFormats_Associations_LayerClusterToCaloParticleAssociator_h
+#define SimDataFormats_Associations_LayerClusterToCaloParticleAssociator_h
+// Original Author:  Marco Rovere
+
+// system include files
+#include <memory>
+
+// user include files
+
+#include "SimDataFormats/Associations/interface/LayerClusterToCaloParticleAssociatorBaseImpl.h"
+
+// forward declarations
+
+namespace hgcal {
+
+  class LayerClusterToCaloParticleAssociator {
+  public:
+#ifndef __GCCXML__
```

Comment:
i don't think this is necessary any more

Reply:
> i don't think this is necessary any more

Correct.